packages/design-system/src/styles/components/_UsaBanner.scss

```diff
@@ -1,189 +1,208 @@
-    display: block;
+  @media (min-width: $media-width-md) {
+    align-items: center;
+    // Changing display:grid to flex for larger viewports
+    // The button goes from being a full-width element on mobile/tablet to something that looks like it's an inline link with text and needs to flow appropriately. Flex accomplishes this.
+    display: flex;
+    min-height: auto;
   }
 }
 
-.ds-c-usa-banner__collapse-banner-container {
-  align-items: center;
-  background-color: var(--usa-banner-close__background-color);
-  display: flex;
-  height: $spacer-6;
-  justify-content: center;
-  margin-block-start: calc(-1 * #{$spacer-1});
-  margin-inline: auto calc(-1 * #{$spacer-2});
-  width: $spacer-6;
-
-  .ds-c-icon {
-    height: var(--font-size-lg);
-    width: var(--font-size-lg);
-  }
+.ds-c-usa-banner__header-icon {
+  grid-area: icon;
+  height: 1em;
+  width: 1em;
 }
 
-.ds-c-usa-banner__header-flag {
-  height: 11px;
-  margin-inline-end: $spacer-1;
-  width: $spacer-2;
+.ds-c-usa-banner__header-text {
+  grid-area: text;
+  margin: 0;
 }
 
-.ds-c-usa-banner__header-text {
-  display: inline;
+.ds-c-usa-banner__action {
+  color: var(--usa-banner-link__color);
+  grid-area: action;
   margin: 0;
-  span {
-    margin-inline-end: $spacer-1;
+  text-decoration: underline;
+
+  .ds-c-usa-banner__header--expanded & {
+    // When banner is expanded on mobile/tablet, action text goes away
+    display: none;
   }
 }
 
-.ds-c-usa-banner__header-action {
-  color: var(--usa-banner-action__color);
-  display: block;
-  margin-block: 2px 0;
-  text-decoration: underline;
+.ds-c-usa-banner__action-icon {
+  color: currentColor;
+  height: 0.75em;
+  margin-inline-start: 0.5ch;
+  // Needed to overwrite the vertical alignment of the icon so the action text appears aligned with the banner text. Applied utility flex classes on the action text to re-implement vertical alignment.
+  vertical-align: initial;
+  width: 0.75em;
 }
 
 .ds-c-usa-banner__button {
   background-color: transparent;
   border: 0;
-  bottom: 0;
-  box-shadow: none;
-  margin: 0;
-  padding: 0;
-  text-align: start;
+  color: var(--usa-banner-action__color);
+  cursor: pointer;
+  inset: 0;
+  inset-inline-start: initial;
+  position: absolute;
+  // Mobile/tablet, button is full-size of banner
+  width: 100%;
 
   &:focus {
     @include mixins.focus-styles;
   }
+
+  @media (min-width: $media-width-md) {
+    display: inline-block;
+    height: fit-content;
+    line-height: 1.1;
+    padding: 0;
+    position: relative;
+    text-align: start;
+    width: fit-content;
+  }
 }
 
 .ds-c-usa-banner__button-text {
-  @extend .ds-c-link;
+  // This element is screen-reader only on mobile/tablet
+  @extend .ds-u-visibility--screen-reader;
+
+  @media (min-width: $media-width-md) {
+    // Need to overwrite specific screen reader only styles so the element becomes visible again on larger viewports
+    // Would be nice to have some more flexibility with our utils:
```

Comment:
I can remove this comment later. I was venting.

Reply:
Hmm, yeah, I see what you mean. This is tricky. I wish it were easy to rename our utilities to give us room for more variations, but at this point it would be a wide-sweeping breaking change. Like `ds-u-min-md-*` vs `ds-u-md-*`. It's also more code in our css bundle